src/logic/wallets/getWeb3.ts

```diff
@@ -116,9 +115,12 @@ export const getProviderInfo = async (web3Provider, providerName = 'Wallet') =>
   }
 }
 
-export const getAddressFromENS = async (name) => {
-  const ens = new ENS(web3)
-  return await ens.resolver(name).addr()
+export const getAddressFromENS = async (name: string) => {
```

Comment:
Lets also include return type here

Reply:
@nicosampler please check this

Reply:
It's inferred. Remember that TS can infer types, so if `getAddress` from `web3.eth.ens` returns a `Promise<string>` and  `getAddressFromENS` returns the call to `getAddress` TS can infer the type.
![image](https://user-images.githubusercontent.com/4376509/83141308-d3c31380-a0c5-11ea-92cd-c99212cc371e.png)

![image](https://user-images.githubusercontent.com/4376509/83141179-99f20d00-a0c5-11ea-88fa-f7e007454553.png)